### PR TITLE
Update pagination component

### LINKED ISSUE #490: Improve pagination component behaviour
The design has recently specified some additional behaviours for the pagination component. For example, how it behaves with only 2,3,4 items and so on. Let's make sure we follow those guidelines

### PR DESCRIPTION
**Pull request by drillprop**
Fix #490 
Figma: https://www.figma.com/file/rBjfm2tJ2Pr9M0UNTduU9v/Joystream-Atlas?node-id=6530%3A2471

### CODE REVIEW

File: src/shared/components/Pagination/Pagination.style.tsx [hunk 1]
```diff
@@ -46,7 +46,7 @@ export const ThreeDotsWrapper = styled.div`
 `
 
 const buttonActiveState = css`
-  background-color: ${colors.transparentPrimary[6]};
+  background-color: ${colors.transparentPrimary[12]};
```
Comment (drillprop) [inline src/shared/components/Pagination/Pagination.style.tsx:49]: In Figma this background is set to `transparentPrimary[10]`, but I think it's still too dark, `transparentPrimary[12]` looks slightly better here for my taste

**Review by mikkio-j — APPROVED** [review]
Logic wise looks good and works as expected. For the colors - we should follow designs, but in the other hand the change is small and it's easier to notice which page we have opened, so I approve this

**Review by kdembler — COMMENTED** [review]
Looks good in general but I found some cases where there are jumps in displayed number of items, I don't think that's desirable


https://user-images.githubusercontent.com/12646744/119146471-f246bb00-ba4a-11eb-82e8-dec1f7ac554b.mov


https://user-images.githubusercontent.com/12646744/119146476-f4a91500-ba4a-11eb-9bff-fd04b018c536.mov

**Comment by drillprop** [comment]
> Looks good in general but I found some cases where there are jumps in displayed number of items, I don't think that's desirable

Should be good now. 
I also fixed problems with the storybook. I had to update some imports to make this work, so some changes may look odd in this PR. Unfortunately deployed version for some reason still not working 🤔

File: src/components/VideoGallery.tsx [hunk 2]
```diff
@@ -74,6 +75,7 @@ const VideoGallery: React.FC<VideoGalleryProps> = ({
   }))
   const createClickHandler = (id?: string) => () => id && onVideoClick && onVideoClick(id)
   const createRemoveButtonClickHandler = (id?: string) => () => id && onRemoveButtonClick && onRemoveButtonClick(id)
+  console.log(breakpoints)
```
Comment (kdembler) [inline src/components/VideoGallery.tsx:78]:
```suggestion
```

**Review by kdembler — APPROVED** [review]

**Pull request merged**